inference-engine/tests/unit/vpu/base/graph_transformer_tests.cpp

```diff
@@ -146,9 +150,28 @@ void TestModel::createOutputs(std::vector<DataDesc> outputDescs) {
     }
 }
 
+void TestModel::createInputs(int numInputs) {
+    _model->attrs().set<int>("numInputs", numInputs);
+    _inputs.resize(numInputs);
+
+    for (int i = 0; i < numInputs; ++i) {
+        _inputs[i] = _model->addInputData(formatString("Input %d", i), _dataDesc);
+    }
+}
+
+void TestModel::createOutputs(int numOutputs) {
+    _model->attrs().set<int>("numOutputs", numOutputs);
+    _outputs.resize(numOutputs);
+
+    for (int i = 0; i < numOutputs; ++i) {
+        _outputs[i] = _model->addOutputData(formatString("Output %d", i), _dataDesc);
+    }
+}
+
 Stage TestModel::addStage(
         std::initializer_list<InputInfo> curInputInfos,
-        std::initializer_list<OutputInfo> curOutputInfos) {
+        std::initializer_list<OutputInfo> curOutputInfos,
+        int old_test) {
```

Comment:
What does it mean `old_test`? Could you give a proper name for this flag parameter? It should reflect the real action it makes. Also set its type as `bool`